### ISSUE TITLE
why my experiment results can't reach the original performance?

### ISSUE THREAD

**Issue by JaminLi** [rahulvigneswaran/Lottery-Ticket-Hypothesis-in-Pytorch#7]
Hi, thanks for your nice repo!
When I used your method for my experiment, I found my experiment results can't reach the original performance whether the first iteration or the 35th iteration. 

In <main.py>
#Freezing Pruned weights by making their gradients Zero
grad_tensor = np.where(tensor < EPS, 0, grad_tensor)

Does this issue?

**Comment by l1teng** [comment]
In my implementation, I refactored all of the NumPy operations to the PyTorch ones. It worked, and the accuracy can reach training from scratch.
